tools/VersionController/Program.cs

```diff
@@ -182,7 +185,11 @@ private static void BumpVersions()
                         _minimalVersion.Add(cols[0], new AzurePSVersion(cols[1]));
                     }
                 }
+
+                // Clean MinimalVersion.csv
+                File.WriteAllLines(Path.Combine(_rootDirectory, @"tools\VersionController", "MinimalVersion.csv"), new string[]{ header});
```

Comment:
can we just use `miniVersionFile` here?

Reply:
miniVersionFile points out the one in artifact folder